packages/perf/lib/index.js

```diff
@@ -47,12 +50,41 @@ class FirebasePerfModule extends FirebaseModule {
   constructor(...args) {
     super(...args);
     this._isPerformanceCollectionEnabled = this.native.isPerformanceCollectionEnabled;
+    this._instrumentationEnabled = true;
```

Comment:
If platform is iOS this has to call through to the "I think we should export it" `this.native.isInstrumentationEnabled` ?